vignettes/estimate_infections_workflow.Rmd

```diff
@@ -27,7 +27,7 @@ Alternatively, they can be constructed from individual-level data, for example u
 An example data set called `example_confirm`  is included in the package:
 
 
-```r
+``` r
```

Comment:
do you know why these spaces are inserted everywhere? Is this down to the `knitr` version or something else?

Reply:
Probably related to this change in knitr: https://github.com/yihui/knitr/blob/master/NEWS.md#major-changes